### ISSUE TITLE
Torrent

### ISSUE THREAD

**Issue by expandables2** [chrippa/livestreamer#912]
Hi can livestreamer play torrent files?

**Comment by chrippa** [comment]
No, supported streaming protocols can be found [here](http://docs.livestreamer.io/cli.html#supported-streaming-protocols).
